src/css/main.less

```diff
@@ -472,20 +473,22 @@ fieldset[disabled] .btn-info.active {
 }
 
 .nav-tabs > li > a {
-  color: #8e929b;
+  color: @navbar-inactive-color;
   border-top: none;
   border-left: none;
   border-right: none;
   border-radius: 0;
+  min-width: @navbar-min-width;
```

Comment:
How about
```
min-width: @container-fluid-padding * 6;
```

Consider to rename @container-fluid-padding to some more generic.